x-pack/plugins/apm/public/components/shared/TransactionBreakdown/TransactionBreakdownGraph/index.tsx

```diff
@@ -38,13 +42,23 @@ function TransactionBreakdownGraph(props: Props) {
     [trackApmEvent]
   );
 
+  const isTimeseriesEmpty = isEmpty(timeseries);
```

Comment:
What do you think about handling isEmpty/noHits in one place, in the `TransactionBreakdown` component? We already have similar logic there to handle no data (for the list of KPIs). It could be a prop passed to TransactionBreakdownGraph.